pkg/controller/node/nodecontroller.go

```diff
@@ -203,6 +211,39 @@ func (nc *NodeController) getCondition(status *api.NodeStatus, conditionType api
 // not reachable for a long period of time.
 func (nc *NodeController) monitorNodeStatus() error {
 	nodes, err := nc.kubeClient.Nodes().List(labels.Everything(), fields.Everything())
+	// We assume that only state in which knownNodeSet is empty is during cluster turnup
```

Comment:
Right. Short of different log messages. It was useful when debugging tests, but you're right that it makes no sense now.